### PR TITLE
Update the config file path and check for master password - Closes #66

### LINKED ISSUE #66: Update the path and filename for config migration script
As we moved the configuration migration script to core with https://github.com/LiskHQ/lisk/issues/2046 so we have to update the new path name for the file in `installLisk.sh` file. 

The new file path for that script is `scripts/update_config.js` You ca see more details on https://github.com/LiskHQ/lisk/pull/2048

Also the parameters are different, instead of optional params now we pass command arguments. 

```
node update_config.js <old_config_path> <new_config_path>
```

### PR DESCRIPTION
**Pull request by nazarhussain**
Closes #66 , #74

### CODE REVIEW

File: downloaded/installLisk.sh [hunk 1]
```diff
@@ -265,7 +265,11 @@ upgrade_lisk() {
 	fi
 
 	echo -e "\\nCopying config.json entries from previous installation"
-	"$LISK_INSTALL"/bin/node "$LISK_INSTALL"/updateConfig.js -o "$LISK_BACKUP"/config.json -n "$LISK_INSTALL"/config.json
+    if [[ -z "${LISK_MASTER_PASSWORD}" ]]; then
```
Comment (fchavant) [inline downloaded/installLisk.sh]: Needs to be made backwards compatible.

Comment (nazarhussain) [reply downloaded/installLisk.sh]: @fchavant As per my knowledge and @Nazgolze suggested the branch `0.2.0` is only for latest migration. If we need backward compatible to upgrade between 0.9.x versions, we should keep it on seperate branch.

Comment (fchavant) [reply downloaded/installLisk.sh]: Scripts from the `downloaded` directory are made available on downloads.lisk.io without versioning thus I would keep them as backwards compatible as possible.

**Comment by Nazgolze** [comment]
Yes. And it's gonna break compatibility with 0.9, which we have to care about with this one file because we don't have separate 0.9 and 1.0 installLisk.sh files on downloads.lisk.io

File: downloaded/installLisk.sh [hunk 2]
```diff
@@ -265,7 +265,15 @@ upgrade_lisk() {
 	fi
 
 	echo -e "\\nCopying config.json entries from previous installation"
-	"$LISK_INSTALL"/bin/node "$LISK_INSTALL"/updateConfig.js -o "$LISK_BACKUP"/config.json -n "$LISK_INSTALL"/config.json
+	# If its 0.9.x version
```
Comment (fchavant) [inline downloaded/installLisk.sh]: Typo.

File: downloaded/installLisk.sh [hunk 3]
```diff
@@ -265,7 +265,15 @@ upgrade_lisk() {
...
+	if [ -f "${LISK_INSTALL}/updateConfig.js" ]; then
+		"$LISK_INSTALL"/bin/node "$LISK_INSTALL"/updateConfig.js -o "$LISK_BACKUP"/config.json -n "$LISK_INSTALL"/config.json
+	# If its a latest version
+	elif [[ -z "${LISK_MASTER_PASSWORD}" ]]; then
```
Comment (fchavant) [inline downloaded/installLisk.sh]: Let's chose either `[` or `[[` and consistently use it.

**Pull request merged**